doc/source/notebooks/pandas.ipynb

```diff
@@ -1,2066 +1,2071 @@
-                            "      <th>1</th>\n",
-                            "      <td>3</td>\n",
-                            "    </tr>\n",
-                            "    <tr>\n",
-                            "      <th>2</th>\n",
-                            "      <td>5</td>\n",
-                            "    </tr>\n",
-                            "    <tr>\n",
-                            "      <th>3</th>\n",
-                            "      <td>7</td>\n",
-                            "    </tr>\n",
-                            "    <tr>\n",
-                            "      <th>4</th>\n",
-                            "      <td>9</td>\n",
-                            "    </tr>\n",
-                            "  </tbody>\n",
-                            "</table>\n",
-                            "</div>"
-                        ],
-                        "text/plain": [
-                            "   val\n",
-                            "0    1\n",
-                            "1    3\n",
-                            "2    5\n",
-                            "3    7\n",
-                            "4    9"
-                        ]
-                    },
-                    "execution_count": 34,
-                    "metadata": {},
-                    "output_type": "execute_result"
-                }
-            ],
-            "source": [
-                "num_1 = pd.DataFrame({\"val\": [1, 3, 5, 7, 9]})\n",
-                "num_2 = pd.DataFrame({\"val\": [2, 4, 6, 8, 10]})\n",
-                "\n",
-                "num_1[num_2[\"val\"] % 2 == 0]  # Even numbers?"
-            ]
-        },
-        {
-            "cell_type": "markdown",
-            "metadata": {},
-            "source": [
-                "To avoid such kind of misuse, in GreenplumPython, it is impossible to refer to other `DataFrame`s except for the \"current\" one in an expression except when using the `in_()` expression.\n",
-                "\n",
-                "This is because GreenplumPython only accepts a `Callable` as argument for expression and will bind it to the current `DataFrame` automatically. "
-            ]
-        }
-    ],
-    "metadata": {
-        "kernelspec": {
-            "display_name": "Python 3 (ipykernel)",
-            "language": "python",
-            "name": "python3"
-        },
-        "language_info": {
-            "codemirror_mode": {
-                "name": "ipython",
-                "version": 3
-            },
-            "file_extension": ".py",
-            "mimetype": "text/x-python",
-            "name": "python",
-            "nbconvert_exporter": "python",
-            "pygments_lexer": "ipython3",
-            "version": "3.9.13"
-        },
-        "vscode": {
-            "interpreter": {
-                "hash": "aee8b7b246df8f9039afb4144a1f6fd8d2ca17a180786b69acc140d282b71a49"
-            }
-        }
-    },
-    "nbformat": 4,
-    "nbformat_minor": 4
+ "cells": [
```

Comment:
Did this file actually change or only formatting? I would just skip this one if so.